Source/MediaInfo/Video/File_Vc3.cpp

```diff
@@ -710,7 +710,7 @@ void File_Vc3::Data_Parse()
         }
         else
         {
-            FrameInfo.PTS=FrameInfo.DTS=FrameInfo.DUR=(int64u)-1;

```

Comment:
Why? Not same assignment (PTS, DTS, DUR are different)

Reply:
error :-(

Reply:
revert